targets/pna_nic/tests/Makefile.am

```diff
@@ -0,0 +1,27 @@
+SUBDIRS = .
+
+AM_CPPFLAGS += \
+-isystem $(top_srcdir)/third_party/gtest/include \
+-I$(srcdir)/.. \
+-I$(srcdir)/ \
+-DTESTDATADIR=\"$(srcdir)/testdata\"
+LDADD = $(builddir)/../libpnanic.la \
+$(top_builddir)/third_party/gtest/libgtest.la \
+$(top_builddir)/src/bm_apps/libbmapps.la \
+-lboost_filesystem
+
+# Define unit tests
+common_source = main.cpp ../../test_utils/utils.cpp ../../test_utils/utils.h
```

Comment:
I haven't used autotool / automake in a while, but I think best practice would be to package utils.cpp as a convenience library by adding it to `check_LTLIBRARIES`. See https://www.gnu.org/software/automake/manual/html_node/Libtool-Convenience-Libraries.html.

Reply:
Ok. Thanks for the advice.